plugin/src/test/java/org/opensearch/ml/rest/RestMLRegisterModelActionTests.java

```diff
@@ -136,6 +137,8 @@ public void testRegisterModelRequest() throws Exception {
         assertEquals("test_model", registerModelInput.getModelName());
         assertEquals("1", registerModelInput.getVersion());
         assertEquals("TORCH_SCRIPT", registerModelInput.getModelFormat().toString());
+        assertEquals("query: ", ((TextEmbeddingModelConfig) registerModelInput.getModelConfig()).getQueryPrefix());
+        assertEquals("passage: ", ((TextEmbeddingModelConfig) registerModelInput.getModelConfig()).getPassagePrefix());
```

Comment:
can I get a test of null-ness if these are not specified as well?

Reply:
Okay.